# Message header
Header header

# The steering angle error returned by the lane detection
float64 steering_error

# Whether the vehicle is at a curve or on a straight lane
bool at_curve
